src/scripts/main.js

```diff
@@ -1,3 +1,422 @@
+            +[...b.children][idx].textContent.replace(/[^0-9.]/g, '')
+          ) {
+            return -1;
+          }
+
+          return 0;
+        });
+
+        elem.classList.remove('DESC');
+        elem.classList.add('ASC');
+      } else {
+        sortedRows = [...table.tBodies[0].children].sort((a, b) => {
+          if (
+            +[...a.children][idx].textContent.replace(/[^0-9.]/g, '') <
+            +[...b.children][idx].textContent.replace(/[^0-9.]/g, '')
+          ) {
+            return 1;
+          }
+
+          if (
+            +[...a.children][idx].textContent.replace(/[^0-9.]/g, '') >
+            +[...b.children][idx].textContent.replace(/[^0-9.]/g, '')
+          ) {
+            return -1;
+          }
+
+          return 0;
+        });
+
+        elem.classList.remove('ASC');
+        elem.classList.add('DESC');
+      }
+
+      break;
+
+    default:
+      if (elem.className === '' || elem.className === 'DESC') {
+        sortedRows = [...table.tBodies[0].children].sort((a, b) => {
+          return [...a.children][idx].textContent.localeCompare(
+            [...b.children][idx].textContent,
+          );
+        });
+        elem.classList.remove('DESC');
+        elem.classList.add('ASC');
+      } else {
+        sortedRows = [...table.tBodies[0].children].sort((a, b) => {
+          return [...b.children][idx].textContent.localeCompare(
+            [...a.children][idx].textContent,
+          );
+        });
+        elem.classList.remove('ASC');
+        elem.classList.add('DESC');
+      }
+
+      break;
+  }
+
+  table.tBodies[0].remove();
+  insertRows(sortedRows);
+  removeCellText();
+}
+
+function insertRows(rows) {
+  const tBody = document.createElement('tbody');
+
+  rows.map((row) => {
+    tBody.append(row);
+  });
+
+  table.append(tBody);
+  selectRow();
+}
+
+function selectRow() {
+  table.tBodies[0].addEventListener('click', (e) => {
+    e.target.parentNode.classList.toggle('active');
+
+    [...document.querySelector('tbody').children].map((row) => {
+      if (row !== e.target.parentNode) {
+        row.classList.remove('active');
+      }
+    });
+  });
+}
+
+function createForm() {
+  const form = document.createElement('form');
+
+  form.classList.add('new-employee-form');
+
+  const headlines = [...table.tHead.firstElementChild.children].map(
+    (th) => th.textContent,
+  );
+
+  headlines.forEach((headline) => {
+    const label = document.createElement('label');
+
+    label.textContent = `${headline}:`;
+
+    form.append(label);
+  });
+
+  form.append(createButton());
+
+  document.body.appendChild(form);
+
+  addInputs();
+}
+
+createForm();
+
+function createButton() {
+  const button = document.createElement('button');
+
+  button.setAttribute('type', 'submit');
+  button.textContent = 'Save to table';
+
+  return button;
+}
+
+function createSelect() {
+  const select = document.createElement('select');
+
+  const cities = [...table.tBodies[0].children]
```

Comment:
It's better to have defined constant list of cities

Reply:
constant list of cities is defined